README.md

```diff
@@ -364,8 +364,10 @@ sign_up.is_a?(SignUp) # => true
 This is basically like [`ActiveRecord#becomes`](http://apidock.com/rails/v4.2.1/ActiveRecord/Persistence/becomes), but with less bugs and more consistent behavior.
 
 **Note that `cast` is destructive.** The originally casted record (`user`) and the returned record (`sign_up`)
-share internal state (such as attributes). To avoid unexpected behavior, do not use the original record
-after casting it.
+share internal state (such as attributes). To avoid unexpected behavior, the original record will raise an error when trying to change or persist it. Also, casting of a record that has changes in its loaded associations is prevented, because those changes will not be known to the record that is created while casting.  
```

Comment:
A bit hard to read, I suggest "... because those changes would be lost".

Reply:
Thanks for the suggestion!